pkg/kubelet/apis/well_known_labels.go

```diff
@@ -24,8 +24,10 @@ const (
 
 	LabelInstanceType = "beta.kubernetes.io/instance-type"
 
-	LabelOS   = "beta.kubernetes.io/os"
-	LabelArch = "beta.kubernetes.io/arch"
+	LabelArch      = "beta.kubernetes.io/arch"
```

Comment:
Why are these labels vs `nodeInfo` entries?

Especially the versions don't make sense as labels for the same reason I posted in https://github.com/kubernetes/kubernetes/pull/51006#issuecomment-327251049